File: LeetCode_26/Solution_ZhongJing.py
from typing import List


class Solution:
    """
    日期：2023-08-04
    作者：仲景
    """
    
    def removeDuplicates(self, nums: List[int]) -> int:
        index = 0
        for i in range(len(nums)):
            if nums[i] != nums[index]:
                index += 1
                nums[index] = nums[i]
        
        return index + 1

File: LeetCode_26/Solution_ZhongJing.go
package main

// 日期：2023-07-1769_01
// 作者：仲景
func removeDuplicates(nums []int) int {
	index, numsLen := 1, len(nums)
	if numsLen == 0 {
		return 0
	}
	for i := 1; i < len(nums); i++ {
		if nums[i] != nums[i-1] {
			nums[index] = nums[i]
			index++
		}
	}
	return index
}
